Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -91,20 +91,42 @@ public static bool IsBuildingOpen(DFLocation.BuildingTypes buildingType)
                     closeHours[(int)buildingType] > DaggerfallUnity.Instance.WorldTime.Now.Hour);
         }
 
-        // Allow mods to register custom model activation methods.
-        public delegate void ModelActivation(Transform transform);
-        private static Dictionary<string, ModelActivation> customModelActivations = new Dictionary<string, ModelActivation>();
+        // Allow mods to register custom flat / model activation methods.
+        public delegate void CustomActivation(Transform transform);
+        private static Dictionary<string, CustomActivation> customActivations = new Dictionary<string, CustomActivation>();
 
-        public static bool RegisterModelActivation(uint modelID, ModelActivation modelActivation)
+        public static void RegisterCustomActivation(uint modelID, CustomActivation customActivation)
```

Comment:
When i updated formula overrides i added a `Mod` parameter to [enforce load order](https://github.com/Interkarma/daggerfall-unity/blob/43e9f3cf4ed15d4a2079e65e03f7275d52e2be31/Assets/Scripts/Game/Formulas/FormulaHelper.cs#L2362) instead of always replacing/rejecting previously set delegates (also see [this issue](https://github.com/Interkarma/daggerfall-unity/issues/1668#issuecomment-589084075)). Might be worth to discuss if is also useful here.

Reply:
I'm all for consistency so I will implement this and test before committing. 

If this feature becomes popular and mods start overriding each other, clear feedback will be important to know which mod registered what so I would like to add a Debug.Log during an overwrite if that's OK with everyone.

Reply:
I have implemented this and first tests show it works as intended, any mod that is loaded after another will be allowed to overwrite any existing custom activation. I'll set up a few more tests where I have the mods hook in at different points of code execution.